packages/protocol/test/L2/V1TaikoL2.test.ts

```diff
@@ -1,9 +1,11 @@
 import { expect } from "chai"
 import { ethers } from "hardhat"
+import { V1TaikoL2 } from "../../typechain"
 
 describe("V1TaikoL2", function () {
```

Comment:
```suggestion
describe("TaikoL2", function () {
```
this contract has been renamed to `TaikoL2`